demos/collection/grid.php

```diff
@@ -72,6 +74,45 @@
 // $grid->addExecutorButton($deleteExecutor, new Button(['icon' => 'times circle outline']));
 
 $sel = $grid->addSelection();
+// Executing a modal on a bulk selection
+$callback = function ($m, $ids) use ($grid) {
+    if (!$ids) {
+        Message::addTo($m, [
```

Comment:
Build action button should be disabled instead when nothing is selected.

Reply:
nice idea. have you an example of such magic?

Reply:
Maybe the master checkbox code could enable/disable bulk action button by querying a dedicated class

Reply:
Wait on #1921 in the meantime, the action will return a void array, then iterating through it will do nothing.